.github/workflows/sustain-bugs.yml

```diff
@@ -0,0 +1,23 @@
+name: sustain
+
+on:
+  issues:
+    types: [labeled]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env: 
+      PROJECT_COLUMN_ID: ${{secrets.PROJECT_COLUMN_ID}}
+    steps:
+    - name: Maybe move an issue to the project board
+      uses: actions/github-script@0.4.0
```

Comment:
Should this use the latest version?

Reply:
I suppose so... I have not tested on `0.8.0`, but it should be fine 👍

Reply:
Updated in ae2e0617a

Reply:
Tested - works as expected